.travis.yml

```diff
@@ -11,9 +11,6 @@ node_js:
 cache:
   directories:
   - node_modules
-env:
-  global:
-  - NODE_ENV=production
 install:
 - npm --production=false install
 - npm --production=false update
```

Comment:
We can simplify this and remove `--production=false` if we're not setting `NODE_ENV=production` globally.

Reply:
Fixed in 8ded00f